src/cbmc/cbmc_parse_options.cpp

```diff
@@ -30,32 +30,32 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <cpp/cprover_library.h>
 
+#include <goto-instrument/cover.h>
+#include <goto-instrument/full_slicer.h>
+#include <goto-instrument/nondet_static.h>
+#include <goto-instrument/reachability_slicer.h>
```

Comment:
Nit pick: I'd prefer for the `goto-instrument` includes to be separated by a blank line and kept after the `goto-programs` includes. Rationale: 1) `goto-programs` is more fundamental than `goto-instrument`; 2) if there are any missing includes in some header in `goto-programs` then it must not be the case that some header in `goto-instrument` incidentally provides such an include, and thus the issue is left unnoticed.

Reply:
Noticed the implications of (2). Done.